native/dlaqr04.go

```diff
@@ -419,9 +419,6 @@ func (impl Implementation) Dlaqr04(wantt, wantz bool, n, ilo, ihi int, h []float
 			// Shuffle shifts into pairs of real shifts and pairs of
 			// complex conjugate shifts using the fact that complex
 			// conjugate shifts are already adjacent to one another.
-			// TODO(vladimir-ch): Dlaqr5 also does this shuffling.
```

Comment:
In fact, the shuffling here could _probably_ be removed but I'm not sure right now and it's safer to leave it.